### ISSUE TITLE
How to implement pagination

### ISSUE THREAD

**Issue by arukumar** [ericgio/react-bootstrap-typeahead#49]
I have see the demo of React bootstrap tyeahead, but can not understand the pagination feature. can you please explain about it.

```
<Typeahead  labelKey="name" ref="typeahead"  onChange={(selected)=>this.handleChnage(selected)}  
    onInputChange={(text)=>this.handleInputChange(text)}  placeholder="what are you shopping for today?" 
 options = {TestApp} minLength = {3} 
     renderMenuItemChildren ={(props, option, idx) => { 
         console.log("Options Value",msg)
         let img =   <img  src={'/static/images/'+ option.imageUrl} width='10%' height='10%' style={style} />
         return [          
                 <div key="name">
            {img}{option.name}
            </div>      
         ];
     }}  />
```

**Comment by ericgio** [comment]
I forgot to add documentation for this prop/feature in the readme. Here's the documentation from the code itself:

> For large option sets, initially display a subset of results for improved performance. If users scroll to the end, the last item will be a link to display the next set of results. Value represents the number of results to display. `0` will display all results.

Basically, if you have many (> 500) options displayed in the menu, the browser starts to lag because there are so many DOM nodes. By default, the component will truncate to `200` the number of results displayed in the menu. You can change this number to be more or less, with `0` displaying all the options.

**Comment by arukumar** [comment]
Thanks for replay,
 but how to set  "initialResultCount" ?. As it shows 100 by default.
And as you mentioned that the last item will be a link to display the next set of results, how we will acheive this ? is there any props

**Comment by ericgio** [comment]
https://github.com/ericgio/react-bootstrap-typeahead/blob/master/src/Typeahead.react.js#L265

`this.props.paginateResults` is passed through to `initialResultCount`. So, set `paginateResults` to be whatever number you want. If the number of results exceeds the number set by `paginateResults` (or if you set nothing, if it exceeds 100), the menu will automatically display a menu item to show more results. Note that if the number is not exceeded, the menu item will not be shown.

**Comment by arukumar** [comment]
thanks ericgio

**Comment by arukumar** [comment]
Hi ericgio,
i have implemented the same and it works fine, but can we use custom link in place of "Display Next .. result" like "View All".

**Comment by ericgio** [comment]
Duplicate of https://github.com/ericgio/react-bootstrap-typeahead/issues/47

This is available in v0.8.0 using the `paginationText` prop.

**Comment by arukumar** [comment]
Hi ericgio,
Still i am facing issue.
I have used  paginationText="view All" in typeahead tag, but still it display "Display Additional Result".
can we implement click functionality after setting the text in painationText, so that i can navigate to other page?
